package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/lambda-tools",
-  "version": "3.0.4",
+  "version": "3.1.0",
```

Comment:
I often wait to bump the version of the project until the functional changes are merged so that it avoid possible conflicts with updates while this is still under review

Reply:
Fixed. I reverted this change.

Reply:
Awesome, thank you!